app/src/renderer/vuex/store.js

```diff
@@ -6,7 +6,7 @@
 
 import Vue from "vue"
 import Vuex from "vuex"
-import _ from "lodash"
+import { merge } from "lodash"
```

Comment:
I think you could just import the one function with (didn't test it):
```suggestion
import merge from "lodash/merge"
```

Reply:
I'll try now

Reply:
It works! but I think I like more the `{ }` version, they both import only the merge function thanks to webpack tree shaking